packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -324,7 +326,7 @@ const PRESETS = [
         description: '',
         templateSource: {
             type: TEMPLATE_SOURCE_BUNDLE,
-            id: 'typescript-minimal'
+            id: 'chakra-storefront'
```

Comment:
I believe this was a mistake, the preset is `chakra-storefront-test-project` but we targeted typescript-mininal which has nothing to do with chakra-storefront